### ISSUE TITLE
How to use with React Native 0.59 version

### ISSUE THREAD

**Issue by pie6k** [Microsoft/react-native-code-push#1539]
React Native 0.59 changes the way main bundle file is loaded and I'm not sure how to adjust it to work with code push

see: https://github.com/react-native-community/rn-diff-purge/compare/version/0.58.6...version/0.59.0#diff-ad42fe05482c788c487f5b3a971ce4cbR18

**Comment by fang0rnz** [comment]
Same problem here

**Comment by yuri-kulikov** [comment]
Hi @pie6k,
Thanks for reporting!

You're right! We are currently working on a fix and will release a new version soon.

**Comment by Jekiwijaya** [comment]
```diff
{
#if DEBUG
  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index" fallbackResource:nil];
#else
-  return [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
+ return [CodePush bundleURLForResource:@"main" withExtension:@"jsbundle"];
#endif
}
```

**Comment by cdriscol** [comment]
I fixed this a similar way to @Jekiwijaya , just dropped this line into the `AppDelegate.m`:

```- (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
{
#if DEBUG
  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index" fallbackResource:nil];
#else
  return [CodePush bundleURL];
#endif
}
@end```

**Comment by alexandergoncharov** [comment]
Hi all,

We implemented fix for `link` logic for `react-native@0.59` and released new version of CodePush plugin: https://github.com/Microsoft/react-native-code-push/releases
So, I'm going to close this issue for now but please feel free to reopen it if you have any questions or face this issue again.

Thanks,
Alexander

**Comment by zheldRnaskan** [comment]
Hi all, I update react-native-code-push latest version and also did solutions as mentioned above but still have a problem. I think local files in my app are not properly implemented on js file that code-push sent to my app.

when I installed my fresh app and open webview(that is rendered from local static html file) it worked but, but after OTA updated via code-push, I got error "Domain: kCFErrorDomainCFNetwork Error Code: 1" on my webview.
